edi_punchout/views/templates.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Hunki Enterprises BV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <template id="ids_send_cart">
+        <Warenkorb xmlns="http://www.itek.de/Shop-Anbindung/Warenkorb/">
```

Comment:
Why is this hard coded to this URL?
I don't see this view appear in runboat. Wher is it supposed to be?

Reply:
that's what the [docs](https://www.itek.de/fileadmin/user_upload/itek-website/Beratung/Standardisierung/IDS_Connect/2.5/20201225_IDS_Schnittstelle_2_5.pdf) say the correct namespace for this kind of xml is. And you won't see this view in runboat, this is what generates the xml being sent to the webshop when using the ids protocol.

Reply:
Thanks, makes sense.